doc/source/advanced.md

```diff
@@ -39,7 +39,45 @@ We recommend the community-maintained [nginx-ingress](https://github.com/helm/ch
 controller, [**kubernetes/ingress-nginx**](https://github.com/kubernetes/ingress-nginx).
 Note that Nginx maintains two additional ingress controllers.
 For most use cases, we recommend the community maintained **kubernetes/ingress-nginx** since that
-is the ingress controller that the development team has the most experience using.
+is the ingress controller that the development team has the most experience using. See below for
+**GCE Ingress** settings.
+
+#### GCE Ingress and manual HTTPS
+To set up HTTPS with GCE Ingress, you must:
+1. offload the proxy, enable `https` (and ideally, change it from 
+ external `LoadBalancer` to close it from outside world - we have ingress for that)
+2. create TLS secrets same as for other Ingresses (e.g. [nginx](https://kubernetes.github.io/ingress-nginx/user-guide/tls/) )
+3. reference the secret in the ingress spec and add `pathSuffix: *`
+
+```yaml
+proxy:
+  https:
+    enabled: true
+    type: offload
+  service:
+    type: NodePort
+ingress:
+  enabled: true
+  annotations:
+    # kubernetes.io/ingress.global-static-ip-name: YOUR-STATIC-IP-NAME
+    kubernetes.io/ingress.class: gce
+    kubernetes.io/ingress.allow-http: "false"
+  hosts:
+  - example.com  # change it to you domain
+  pathSuffix: "*"
```

Comment:
I learned:

https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/03127a9e351ed3ffca99969ef6f9a2b115b46516/jupyterhub/templates/ingress.yaml#L14-L24

So, this will influence the rule of what to match, by saying: match everything. Hmmm, this is the default behavior for ingress-nginx I believe. I mean, I arrive at the destination even though i request `https://myhubdomain.com/hub/home` for example. Is this another GCE Ingress Controller vs nginx-ingress-controller deviation?

Reply:
Yes, they behaves differently. See my following pr: https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/988